k8s/namespaces/jenkins/jenkins.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     enable: true
     retry: true
   chart:
-    repository: https://kubernetes-charts.storage.googleapis.com/
+    repository: https://charts.jenkins.io
```

Comment:
prob need to update chart version as well? is 1.16.1 published there?

Reply:
Yes, it is. Want to leave update for some other day.

```
helm search repo jenkins --version 1.16.1

NAME           	CHART VERSION	APP VERSION	DESCRIPTION
jenkins/jenkins	1.16.1       	lts        	Open source continuous integration server. It s...
stable/jenkins 	1.16.1       	lts        	Open source continuous integration server. It s...
```